### ISSUE TITLE
USER STORY: ML Model Building

### ISSUE THREAD

**Issue by SamSswi** [SamSswi/milestone-project-mildew-detection-in-cherry-leaves#2]
As a **client** I want to **build an ML model and generate reports**
  
  ### Acceptance Criteria:
- Delivering an ML model with an accuracy of at least 95% on the test set.
- Saved pkl file with the result of the model evaluation
- Saved images of the plots of model history 

  ## Tasks
- Implement Image Augmentation
- Generate Augmented Images for the Training set and standardize pixels for the other sets
- ML model creation, training and tuning
- Learning curve plot for model performance save as png
- Model evaluation on test set save as pkl
